src/main/kotlin/org/rust/cargo/runconfig/CargoRunStateBase.kt

```diff
@@ -58,16 +58,16 @@ abstract class CargoRunStateBase(
         return commandLine
     }
 
-    override fun startProcess(): ProcessHandler = startProcess(emulateTerminal = false)
+    override fun startProcess(): ProcessHandler = startProcess(processColors = true)
 
-    fun startProcess(emulateTerminal: Boolean): ProcessHandler {
-        var commandLine = cargo().toColoredCommandLine(environment.project, prepareCommandLine())
-        if (emulateTerminal) {
-            commandLine = PtyCommandLine(commandLine)
-                .withInitialColumns(PtyCommandLine.MAX_COLUMNS)
-                .withConsoleMode(false)
+    fun startProcess(processColors: Boolean): ProcessHandler {
```

Comment:
Let's add a doc comment on what `processColors` means and why and when it's needed